src/CodeGen_GPU_Host.cpp

```diff
@@ -104,7 +104,7 @@ CodeGen_GPU_Host<CodeGen_CPU>::CodeGen_GPU_Host(Target target)
     // earlier ones.
     if (target.has_feature(Target::OpenGL)) {
         debug(1) << "Constructing OpenGL device codegen\n";
-        cgdev[DeviceAPI::GLSL] = new CodeGen_OpenGL_Dev(target);
+        cgdev[DeviceAPI::GLSL] = std::make_unique<CodeGen_OpenGL_Dev>(target);
```

Comment:
Shouldn't all of the subsequent assignments into `cgdev` in this function also use `make_unique` now too?

Reply:
The others return std::unique_ptr now.